3.2/root/usr/bin/run-mongod-replication

```diff
@@ -50,7 +50,7 @@ fi
 cache_container_addr
 mongo_common_args="-f $MONGODB_CONFIG_PATH"
 if ! [[ -v MONGODB_USER && -v MONGODB_PASSWORD && -v MONGODB_DATABASE && -v MONGODB_ADMIN_PASSWORD && -v MONGODB_KEYFILE_VALUE && -v MONGODB_REPLICA_NAME ]]; then
-    usage
+  usage
```

Comment:
This can be applied to all image versions, and don't need to be here, moving it to #201.